DirectProgramming/DPC++FPGA/ReferenceDesigns/gzip/gzip.vcxproj

```diff
@@ -22,8 +22,6 @@
     <ClInclude Include="src\crc32.hpp" />
     <ClInclude Include="src\gzipkernel.hpp" />
     <ClInclude Include="src\kernels.hpp" />
-    <ClInclude Include="src\pipe_array.hpp" />
-    <ClInclude Include="src\pipe_array_internal.hpp" />
```

Comment:
I'll admit I don't understand these vcxproj files, but don't you need to add some reference to the new include file (pipe_utils.hpp)?

Reply:
I'm assuming this is covered by our Windows regression test infra when testing with the .sln files, in which case these tests pass.

Reply:
It is not. This portion of the vcxproj is kind of weird. Any .cpp files in there will be compiled. Any HPP files will show up in the GUI as files "in" the project. So, all this does is remove the HPP files from showing up in the sidebar of VS.